tests/Internal/Codebase/InternalCallMapHandlerTest.php

```diff
@@ -476,12 +404,17 @@ class InternalCallMapHandlerTest extends TestCase
         'cachingiterator::getinneriterator' => ['8.1', '8.2'],
         'callbackfilteriterator::getinneriterator' => ['8.1', '8.2'],
         'curl_multi_getcontent',
-        'datetime::add' => ['8.1', '8.2'],
+        'datetime::add' => ['8.1', '8.2'], // DateTime does not contain static
+        'datetime::modify' => ['8.1', '8.2'], // DateTime does not contain static
+        'datetime::createfromformat' => ['8.1', '8.2'], // DateTime does not contain static
         'datetime::createfromimmutable' => ['8.1'],
         'datetime::createfrominterface',
-        'datetime::setdate' => ['8.1', '8.2'],
-        'datetime::settimezone' => ['8.1', '8.2'],
-        'datetime::sub' => ['8.1', '8.2'],
+        'datetime::setdate' => ['8.1', '8.2'], // DateTime does not contain static
+        'datetime::setisodate' => ['8.1', '8.2'], // DateTime does not contain static
+        'datetime::settime' => ['8.1', '8.2'], // DateTime does not contain static
+        'datetime::settimestamp' => ['8.1', '8.2'], // DateTime does not contain static
+        'datetime::settimezone' => ['8.1', '8.2'], // DateTime does not contain static
+        'datetime::sub' => ['8.1', '8.2'], // DateTime does not contain static
```

Comment:
Hopefully can add a simple comment for each return type error to know why it's blocked.